app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.java

```diff
@@ -32,22 +32,25 @@
 				"bureau_de_change",	"money_transfer", "post_office", "library",
 				"courthouse", "embassy", "car_wash", "car_rental",
 				"marketplace", "fuel", "driving_school", "dentist",
-				"doctors", "clinic", "pharmacy", "veterinary" };
+				"doctors", "clinic", "pharmacy", "veterinary",
+				"place_of_worship", "school", "townhall"};
```

Comment:
You should leave school out. Reasons:
- You cannot simply enter a school (and walk down all the hallways etc) as a stranger.
- The information is not useful: Either the school has facilities or not, if it doesn't, there won't be any pupils in wheelchair. It's not like "hooray, today is my first day in this school! Oh, I can't enter it."